frontend/benefit/handler/src/components/applicationReview/salaryBenefitCalculatorView/SalaryBenefitCalculatorView.tsx

```diff
@@ -372,10 +386,181 @@ const SalaryBenefitCalculatorView: React.FC<
           invalid={!!getErrorMessage(fields.endDate.name)}
           aria-invalid={!!getErrorMessage(fields.endDate.name)}
           errorText={getErrorMessage(fields.endDate.name)}
-          style={{ paddingRight: `${theme.spacing.xs}` }}
+          style={{ paddingRight: `${theme.spacing.s}` }}
         />
       </$GridCell>
 
+      {!isManualCalculator && (
```

Comment:
I would suggest to put it into another component / refactor somehow. This is superlong component already.

Reply:
I moved the manual calculator UI to a separate component